src/config.rs

```diff
@@ -552,6 +598,14 @@ impl Context {
         ensure!(key.starts_with("ui."), "get_ui_config(): prefix missing.");
         self.sql.get_raw_config(key).await
     }
+
+    /// Set synchronised config value on a receiver side if necessary.
+    pub(crate) async fn sync_config(&self, key: &Config, value: &str) -> Result<()> {
```

Comment:
this function is called a single time, and the doc string requires not very telling. 
Better inline this function directly in the sync's receiving of a config value. 
Then there is no need for a docstring or function name at all.

Reply:
I tried to improve the doc comment. It's a separate function because the code for syncing other things is also separated into functions and also i'd avoid bloating the `sync` module as we add synchronisation of more things.

Reply:
adding three lines of code to sync.rs while removing 5 lines of code and docs from config.rs does not sound like bloat.   Having unneccessary indirections decreases readability.  Unless there is a second invocation of a function i don't see the point of complicating reading.